staging/publishing/rules.yaml

```diff
@@ -15,7 +15,7 @@ rules:
       branch: release-1.15
       dir: staging/src/k8s.io/code-generator
     name: release-1.15
-    go: 1.12.5
+    go: 1.12.7
```

Comment:
are these changes needed @nikhita @sttts ? I don't think we've done these before when bumping go versions

Reply:
Yes, these are needed if we are bumping the version in the `release-1.15` branch.

> I don't think we've done these before when bumping go versions

We should have done that. :) Created https://github.com/kubernetes/kubernetes/pull/80151 and https://github.com/kubernetes/publishing-bot/pull/197. 

It doesn't _really_ matter because the go version here is mainly used while testing client-go and there haven't been any particular changes in client-go that are specifically related to features in new go versions, but it's definitely better to bump the version in case there are any bug fixes, etc. :+1: